hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/state/CommonEntityAccessor.java

```diff
@@ -51,4 +51,10 @@ public Optional<Entity> getEntityByEvmAddressAndTimestamp(byte[] addressBytes, f
                 .map(t -> entityRepository.findActiveByEvmAddressAndTimestamp(addressBytes, t))
                 .orElseGet(() -> entityRepository.findByEvmAddressAndDeletedIsFalse(addressBytes));
     }
+
+    public Optional<Entity> getEntityByEvmAddressOrAliasAndTimestamp(byte[] alias, final Optional<Long> timestamp) {
```

Comment:
Probably better to have a consistent naming with overloaded params. Another PR adds a `get(TokenID...)`.

```suggestion
    public Optional<Entity> get(byte[] alias, final Optional<Long> timestamp) {
```

Reply:
I renamed the methods, thanks.